### ISSUE TITLE
Latest GMAIL app crashes and reboots phone

### ISSUE THREAD

**Issue by iTudorS** [checkra1n/BugTracker#1118]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 8
2. On what version of iOS is it?
13.3.1
3. What version of checkra1n are you using, and GUI or CLI?
0.9.8.2 beta GUI and CLI does the same thing
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Linux Ubuntu latest version on Live USB on a Windows computer
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
with original Apple USB cable that comes in the box (normal usb to apple connection on phone)

**What are the steps to reproduce the issue?**
1. Install latest Gmail update
2. Open Gmail, use for like 10-20 seconds
3. Phone freezes for some seconds, then tries to respring, sticks with respring wheel for 1-2 minutes, then completely reboots into NON jailbreak state

**What do you expect, and what is happening instead?**

I Expect GMAIL app to work correctly , instead its crashing the entire phone

**Does the issue also occur if you enable Safe Mode?**

Yes

**Any other info, error logs, screenshots, ...?**

Can't provide LOGS, crash reported/cr4shed doesnt detect this as a crash for some reason

**Comment by steven30303** [comment]
It's one of your tweaks. 
try using it while in safe mode, problem should not occur. (For me it doesn't occur at all.)
use icleaner to disable all tweaks and re-enable them all one by one to find out which one is the culprit

**Comment by iTudorS** [comment]
> It's one of your tweaks.
> try using it while in safe mode, problem should not occur. (For me it doesn't occur at all.)
> use icleaner to disable all tweaks and re-enable them all one by one to find out which one is the culprit

Hi Steven, thanks for your reply!

so far what I tried untill now

-restore rootfs, didnt work
-going into non jb mode, only crashes the app, closes itself and Im able to reopen it so it can crash again in a few seconds
-restoring the phone to 13.3.1 and restoring my batchomatic backup

Only tweak I have that interacts with GMAIL, is GmailMidnight, which doesnt matter if its installed and activated or not, no difference

weirdest thing is that this started from nothing, used same setup for a while and one day it just started doing this, without installing any new tweak

I will try today after work jailbreaking into safe mode and see if it still does, will also tru disabling all tweaks with icleaner pro,  so far launching the apo without tweaks with Choicy didnt work, and non of the JB bypass apps work aswell

I will let you know if your solution works for me!

thank you :)

**Comment by iTudorS** [comment]
I tried going into safe mode again, still crashes but doesnt reboot, just crashes the app

disabled all tweaks and everything with icleaner, issue persists, crashes, then reboots completely

**Comment by steven30303** [comment]
Then it might not be jailbreak related (since you mentioned restoring the root fs) and might just be a Gmail problem, did you try to reinstall it? (I know it sounds silly but still)

I myself am on 12.4 and installed it today to check it and it worked fine, might be an ios 13 problem but i can't check that since i have no device on ios 13.

**Comment by iTudorS** [comment]
Yup, reinstalled and nothing, forgot to test it after restoring rootfs without jailbreaking at all, as soon as 13.4 support drops for linux, i'll update and rejb again, probably that will solve it.. for now my fix is turn off auto updates for appstore (appstore++ wont block updates, still updating overnight) and stay on a lower version of gmail...

**Comment by iTudorS** [comment]
I can confirm by now its happening cause of jailbreak
i tried restoring rootfs, and with rootfs restored, gmail runs just fine, restored entire phone, runs fine, then jailbroke, didnt install cydia or anything at all, just jailbroke with checkra1n 0.10 and it crashes again, no tweaks no anything installed, so its cause of checkra1n... hopefully this gets some attention and gets fixed... I really need gmail for work and its annoying... can't downgrade it cause none of the block updates app work, they all update overnight..
